content/releases/prestashop80.md

```diff
@@ -0,0 +1,504 @@
+                                <img loading="lazy" class="image-with-shadow" src="/releases/images/ps8/installer_select_modules.jpeg">
+                            </p>
+                        </div>
+                        <div class="col-md-6">
+                            <p>
+                                <img loading="lazy" class="image-with-shadow" src="/releases/images/ps8/installer_errors.jpeg">
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="section" id="webservice">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-md-4">
+                            <h2 class="section-title">Webservice</h2>
+                            <p>
+                                <strong>The status and URL to your shop’s webservice is now displayed</strong> at the top of the page in <em>Advanced parameters > Webservice</em>. This should be useful to new users and help them troubleshoot common problems.
+                            </p>
+                            <p><strong>*It is now possible to perform partial updates on webservice endpoints using the PATCH method.</strong> This allows integrations to update only part of an entity instead of all the fields at once. <a href="https://github.com/PrestaShop/PrestaShop/pull/27952">Learn more about this change here</a>
+                        </div>
+                        <div class="col-md-8">
+                            <p class="mt-5">
+                                <img loading="lazy" class="image-with-shadow" src="/releases/images/ps8/webservice.jpeg">
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="section" id="media-versioning">
+                <div class="container">
+                    <div class="row">
+                        <div class="col">
+                            <h2 class="section-title">Media versioning</h2>
+                            <p>
+                                <strong>Developers can use media versioning to prevent outdated javascript and stylesheet files from being loaded from the browser’s cache.</strong>
+                            </p>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-4">
+                            <p>A new parameter has been added to <code>registerJavascript</code> and <code>registrerStylesheet</code> methods, allowing module developers to add a version parameter. This parameter will be appended to the asset’s URL, so if the version changes, browsers will download the new asset instead of loading the old version from local cache. <a href="https://github.com/PrestaShop/PrestaShop/pull/24656">Learn more about this change here</a>
```

Comment:
```suggestion
                            <p>A new parameter has been added to <code>registerJavascript</code> and <code>registrerStylesheet</code> methods, allowing module developers to add a version parameter. This parameter will be appended to the asset’s URL, so if the version changes, browsers will download the new asset instead of loading the old version from local cache. <a href="https://github.com/PrestaShop/PrestaShop/pull/24656">Learn more about this change here</a>.
```